examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -1033,6 +1033,7 @@ PROCESS_THREAD(adc_process, ev, data)
 
 		//reading adc value
 		single_adc_sample = AUXADCReadFifo();
+                single_adc_sample = (4300*single_adc_sample)/4096;
```

Comment:
As noted somewhere else as well, you might want to use ti-lib.h instead of the driverlib directly. E.g. ti_lib_aux_adc_read_fifo() instead of AUXADCReadFifo().

But specifically regarding conversion of value there are some functions which might increase accuracy and remove the magic numbers: ti_lib_aux_adc_adjust_value_for_gain_and_offset() and ti_lib_aux_adc_value_to_microvolts().
So maybe something like (consider separating for clarity):
```
single_adc_sample = ti_lib_aux_adc_value_to_microvolts(
      AUXADC_FIXED_REF_VOLTAGE_NORMAL,
      ti_lib_aux_adc_adjust_value_for_gain_and_offset(
          single_adc_sample,
          ti_lib_aux_adc_get_adjustment_gain(AUXADC_REF_FIXED),
          ti_lib_aux_adc_get_adjustment_offset(AUXADC_REF_FIXED))
      );
```